besu/src/main/java/org/hyperledger/besu/cli/subcommands/storage/TrieLogSubCommand.java

```diff
@@ -124,12 +133,67 @@ public void run() {
       final Path dataDirectoryPath =
           Paths.get(
               TrieLogSubCommand.parentCommand.parentCommand.dataDir().toAbsolutePath().toString());
+
+      LOG.info("Estimating trie logs size before pruning...");
```

Comment:
Does this step take long to run? If it's fast I don't see much value in having the log line.

Reply:
think it's valuable, I have seen this vary. Even 10 seconds is a long time for users I think.